src/transformers/utils/import_utils.py

```diff
@@ -400,6 +400,10 @@ def is_apex_available():
     return importlib.util.find_spec("apex") is not None
 
 
+def is_bnb_available():
```

Comment:
A bit too short since we have many of those `is_xxx_available`, can we use ` is_bitsandbytes_available` instead, so that everyone knows immediately what this is?

Reply:
done.